pkgs/desktops/xfce/core/libxfce4windowing/default.nix

```diff
@@ -0,0 +1,15 @@
+{ lib, mkXfceDerivation }:
+
+mkXfceDerivation {
+  category = "xfce";
+  pname = "libxfce4windowing";
+  version = "4.19.2";
```

Comment:
(Not a change request) Actually it already requires [x-d-t 4.19](https://github.com/xfce-mirror/libxfce4windowing/blob/76a10eb67e7741886ba101f3891fca4de062aaac/autogen.sh#L10C8-L10C36) but we did not set `XDT_AUTOGEN_REQUIRED_VERSION` in our setup hook, I guess as long as it configures it should be fine, but otherwise you do need to switch to release tarballs.

---

(Change request) It will still be nice to explicitly list the [build dependencies](https://github.com/xfce-mirror/libxfce4windowing/blob/76a10eb67e7741886ba101f3891fca4de062aaac/configure.ac#L83-L125) in `buildInputs`, and not rely on `wrapGAppsHook` to pull stuff for you ([ref](https://github.com/NixOS/nixpkgs/pull/227825/commits/3fbd18e099f2f619ac3240c803bb0c0264479d3d)).

Reply:
`gtk3` pulls in `glib` and `wayland`, should I add them?

Reply:
> should I add them?

Yeah, that will be nice